amazonfreertossdk/src/main/java/software/amazon/freertos/amazonfreertossdk/AmazonFreeRTOSDevice.java

```diff
@@ -805,21 +814,17 @@ private void subscribeToIoT(final Subscribe subscribe) {
                 mIotMqttManager.subscribeToTopic(topic, qos, new AWSIotMqttNewMessageCallback() {
                     @Override
                     public void onMessageArrived(final String topic, final byte[] data) {
-                        try {
-                            String message = new String(data, "UTF-8");
-                            Log.i(TAG, " Message arrived on topic: " + topic);
-                            Log.v(TAG, "   Message: " + message);
-                            Publish publish = new Publish(
-                                    MQTT_MSG_PUBLISH,
-                                    topic,
-                                    mMessageId,
-                                    QoS,
-                                    data
-                            );
-                            publishToDevice(publish);
-                        } catch (UnsupportedEncodingException e) {
```

Comment:
was this exception removed intentionally?

Reply:
Yes, since I have also changed this String message = new String(data, StandardCharsets.UTF_8); to use the constant and not an arbitrary string. In this was the method does not throw any exceptions anymore